src/config.ts

```diff
@@ -12,6 +12,23 @@ const AWSX_PROFILE_PATH = path.join(AWSX_HOME, 'profiles');
 const AWS_CREDENTIALS_PATH = path.join(AWS_HOME, 'credentials');
 const AWS_CONFIG_PATH = path.join(AWS_HOME, 'config');
 
+const configFileCheck = (): void => {
+  try {
```

Comment:
This try catch block should be moved into app.ts where you call `configFileCheck()`